zapier/introduction/what-is-zapier/zapier-dependencies.md

```diff
@@ -0,0 +1,34 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Zapier Dependencies
+
+---
+## Content
+
+With so many features and possibilities you'd think that Zapier requires many dependencies or a complicated install process. The truth is, using Zapier is as easy as creating an account.
+
+Your first step is going to [Zapier's official registration website](https://zapier.com/sign-up/) followed by creating an account through either Google, Facebook or just email. That's it, you're now ready to use Zapier.
+
+The free version of Zapier is very useful for getting used to the tool and creating some personal integrations, but has its limitations.
```

Comment:
I'd extend this section a bit more. maybe make a bullet point list with the free features. otherwise this insight is basically "create an account"